### PR TITLE
Do less work unboxing

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] if its a big team (i *think*) we can skip a bunch of this stuff (rekey, superseded, etc) cc: @mmaxim  <<<<
- [x] we were unboxing things that we already had unboxed, so the act of selecting a conversation would try and unbox it, over and over

@keybase/react-hackers 
@mmaxim

### CODE REVIEW

File: shared/actions/chat/inbox.js [hunk 1]
```diff
@@ -203,23 +203,27 @@ function _toSupersedeInfo(
...
-  if (supersedes) {
-    yield put(Creators.updateSupersedesState(Map({[conversationIDKey]: supersedes})))
-  }
+  const isBigTeam = c.teamType === ChatTypes.CommonTeamType.complex
 
-  const supersededBy = _toSupersedeInfo(conversationIDKey, c.supersededBy || [])
-  if (supersededBy) {
-    yield put(Creators.updateSupersededByState(Map({[conversationIDKey]: supersededBy})))
-  }
+  if (isBigTeam) {
```
Comment (mmaxim) [inline shared/actions/chat/inbox.js]: `!isBigTeam`?

Comment (mmaxim) [reply shared/actions/chat/inbox.js]: This can be any team, all this stuff is only for KBFS convos.

File: shared/actions/chat/inbox.js [hunk 2]
```diff
@@ -203,23 +203,27 @@ function _toSupersedeInfo(
...
-    yield put(Creators.updateFinalizedState(Map({[conversationIDKey]: c.finalizeInfo})))
+    if (c.finalizeInfo) {
+      yield put(Creators.updateFinalizedState(Map({[conversationIDKey]: c.finalizeInfo})))
+    }
   }
 
   const inboxState = _conversationLocalToInboxState(c)
 
-  if (c && c.snippet) {
+  if (!isBigTeam && c.snippet) {
```
Comment (cjb) [inline shared/actions/chat/inbox.js]: Do you still need to check bare "c" as before?

**Comment by chrisnojima** [comment]
C is checked above right

**Comment by chrisnojima** [comment]
It's set above. If the team type is complex

**Comment by mmaxim** [comment]
I'm wondering if the condition should be reversed.

**Comment by chrisnojima** [comment]
Oh yeah. It's reverse. Cjb can you fix

**Pull request merged**